local-rest-scorer/src/main/java/ai/h2o/mojos/deploy/local/rest/controller/ReadyzApiController.java

```diff
@@ -0,0 +1,34 @@
+package ai.h2o.mojos.deploy.local.rest.controller;
+
+import ai.h2o.mojos.deploy.common.rest.api.ReadyzApi;
+import ai.h2o.mojos.deploy.common.rest.model.ModelSchema;
+import ai.h2o.mojos.deploy.common.transform.MojoScorer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ReadyzApiController implements ReadyzApi {
+
+  private final MojoScorer scorer;
+  private static final Logger log = LoggerFactory.getLogger(ReadyzApiController.class);
+
+  @Autowired
```

Comment:
this can be delete in version of spring that we are using you don't need to put in on constructor.